src/main/java/com/example/school/SchoolApplication.java

```diff
@@ -163,5 +167,65 @@ public static void main(String[] args) {
                 .mark(Optional.empty())
                 .isPresent(false)
                 .build();
+        var mark4 = Activity.builder()
+                .lesson(lesson3)
+                .student(student2)
+                .mark(Optional.empty())
+                .isPresent(false)
+                .build();
```

Comment:
`mark4` and `mark5` are same cases. It means that `student2` skipped 2 times `lesson3`.(It is  impossible) I guess you need to change this case.